source/develop/release-management/features/network/editable-network-names.html.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Editable Network Names
+category: feature
+authors: lgoldber
+wiki_category: Feature
```

Comment:
The wiki_* headers were for traces from the old wiki and were removed. Would you please stop using it?